tck/src/main/java/org/eclipse/microprofile/jwt/tck/util/TokenUtils.java

```diff
@@ -120,16 +112,19 @@ public static String generateTokenString(PrivateKey pk, String kid, String jsonR
             invalidClaims = Collections.emptySet();
         }
         InputStream contentIS = TokenUtils.class.getResourceAsStream(jsonResName);
+        if (contentIS == null) {
+            throw new IllegalStateException("Failed to find resource: " + jsonResName);
+        }
         byte[] tmp = new byte[4096];
```

Comment:
This is a much easier to read way of getting the content of the InputStream in a String

> Scanner s = new Scanner(inputStream).useDelimiter("\\A");
> String result = s.hasNext() ? s.next() : "";